### ISSUE TITLE
Error at startup

### ISSUE THREAD

**Issue by NerdzzyDev** [chziakas/epiphany-ai#1]
When I try to run the test case, I get the error:
```
 python tets_redeval.py
Traceback (most recent call last):
   File "/home/alex/vsprojects/redeval/tets_redeval.py", line 2, in <module>
     from redeval.simulators.performance_simulator import GaslightingSimulator
   File "/home/alex/vsprojects/redeval/.venv/lib/python3.11/site-packages/redeval/simulators/performance_simulator.py", line 5, in <module>
     from epiphany.agents.rag_agent import RagAgent
ModuleNotFoundError: No module named 'epiphany'
```

**Comment by binary-signal** [comment]
Import issue has been resolved in this pull request  #2 
Cheers,

**Comment by chziakas** [comment]
Thank you creating the issue! I fixed the issue, so it should be working now.
